.cbmc-batch/jobs/memcpy_using_uint64/memcpy_using_uint64_harness.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <proof_helpers/utils.h>
+#include <stddef.h>
+
+void *memcpy_impl(void *dst, const void *src, size_t n);
+void *memcpy_using_uint64_impl(void *dst, const void *src, size_t n);
+
+const int MAX = 160;
+/*
+ * Check that the optimized version of memcpy is memory safe
+ * And that it matches the naive version
+ * Coverage 100%
```

Comment:
I thought we wouldn't add coverage as a comment anymore.

Reply:
I think in the particular case where its 100% that's worth noting. But others may disagree.

Reply:
Removed